.env

```diff
@@ -5,25 +5,25 @@ REACT_APP_TRAININGS_URL=
 REACT_APP_DNS=
 REACT_APP_GIT_PATH=
 REACT_APP_PASSWORD=
-REACT_APP_GRAFANA_BASE_URI=
+REACT_APP_MONITORING_BASE_URI=
 # MINIO
 REACT_APP_MINIO_BASE_URI=
 REACT_APP_MINIO_END_POINT=
 REACT_APP_MINIO_PORT=
 REACT_APP_MINIO_END_MINIMUM_DURATION_MS=36000000
 # END POINTS
-REACT_APP_SALON_ROCKETCHAT=
+REACT_APP_SALON_CHAT=
 # VAULT
 REACT_APP_VAULT_BASE_URI=
 REACT_APP_VAULT_KV_ENGINE=onyxia-kv
 # App Env Param
 REACT_APP_INSTANCES_MAX=100
 # Footer
 REACT_APP_ONYXIA_GIT=https://github.com/inseefrlab/onyxia-ui
-REACT_APP_ONYXIA_ROCKETCHAT=
+REACT_APP_ONYXIA_CHAT=
```

Comment:
Not sure why we have 2 env variables for the same info. Can we rationalise this ?

Reply:
I think we only need a REACT_APP_CHAT_URL. I delete the two others references to CHAT, which are not usued yet.